pkg/quarterdeck/api/v1/api.go

```diff
@@ -172,7 +172,8 @@ type APIAuthentication struct {
 }
 
 type RefreshRequest struct {
-	RefreshToken string `json:"refresh_token"`
+	RefreshToken string    `json:"refresh_token"`
+	OrgID        ulid.ULID `json:"org_id,omitempty"`
```

Comment:
For consistency, should the `OrgID` in the Tenant `RefreshRequest` struct also have `omitempty` included?

Reply:
Probably not necessary for functionality since the request doesn't get serialized back into JSON but I do like `omitempty` as a tag for optional parameters, so I'll make the change!

Reply:
I was thinking more for the frontend. I had a brief chat with Cletus and he was wondering for example why he was seeing `next_page_token`, even though it was empty in some responses, but not seeing it included in other responses where a `next_page_token` may be returned. I realized it was because `omitempty` wasn't always listed; so I figured we should try to be more consistent going forward with the optional parameters.